db/migrate/20191028115549_create_application_qualification.rb

```diff
@@ -0,0 +1,15 @@
+class CreateApplicationQualification < ActiveRecord::Migration[6.0]
+  def change
+    create_table :application_qualifications do |t|
+      t.references :application_form, null: false, foreign_key: { on_delete: :cascade }
+
+      t.string :type, null: false
+      t.string :qualification_type, null: false
+      t.string :subject, null: false
+      t.string :grade, null: false
```

Comment:
Do we collect whether or not the grade is predicted?

Reply:
Good point! We've created a `predicted_grade` field that is a boolean now.